# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/browser.ini
# Commit: 11a09ee4618c
# Full Hash: 11a09ee4618cb9432c1d55b67927227f7e69bd97
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-04-16 09:48:46
# Regressor Bug: 1582318
# File Overlap Count: 1
# Description:
#   Bug 1582318: Remove shutting-down processes from pool immediately. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69441
# ==============================================================================

diff -r 48de14388873 -r 11a09ee4618c netwerk/cookie/test/browser/browser.ini
--- a/netwerk/cookie/test/browser/browser.ini	Wed Apr 15 22:33:55 2020 +0000
+++ b/netwerk/cookie/test/browser/browser.ini	Wed Apr 15 22:44:36 2020 +0000
@@ -13,7 +13,6 @@
 [browser_storage.js]
 [browser_serviceWorker.js]
 [browser_sharedWorker.js]
-skip-if = fission #Bug 1582318
 [browser_sameSiteConsole.js]
 support-files = sameSite.sjs
 [browser_oversize.js]
